### PR TITLE
fix(assertions): update fields name and fix layout

### PR DESCRIPTION
**Pull request by neighborhood999**
## Screenshots
截圖 2022-07-06 11 58 20

## Description

- Due to the schema name change, so got the wrong assertions number

Signed-off-by: Jie Peng <im@jiepeng.me>

** PR checklist **

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

### CODE REVIEW

File: static_report/src/components/ComparisonReport/ComparisonReport.tsx [hunk 1]
```diff
@@ -342,7 +342,7 @@ export default function ComparisonReport({
   const { base, input } = data;
   const baseTables = base.tables[reportName];
   const inputTables = input.tables[reportName];
-  const existsDbtTests = (base.tables[reportName] as any)?.dbt_test_results;
+  const existsDbtTests = (base.tables[reportName] as any)?.dbt_test_result;
```
Comment (jonycfu) [inline static_report/src/components/ComparisonReport/ComparisonReport.tsx:345]: why is `any` needed here?

File: static_report/src/components/SingleReport/SingleReport.tsx [hunk 2]
```diff
@@ -60,7 +60,7 @@ export default function SingleReport({ data, name }: Props) {
 
   const overview = getReportAsserationStatusCounts(
     table?.assertion_results
```
Comment (jonycfu) [inline static_report/src/components/SingleReport/SingleReport.tsx:62]: I don't think we should use a ternary operator here anymore `a ? b : c`. 
the method argument should be changed
```ts
export function getReportAsserationStatusCounts(
  assertion: PipeRiderAssertionResult | DbtAssertionResult | null,
): ReportAsserationStatusCounts {  /* .. */ }
```

**Review by jonycfu — CHANGES_REQUESTED** [review]
Root Cause: The higher order components are missing adequate typing, so no compile error was thrown when types changed.

File: static_report/src/components/ComparisonReport/ComparisonReport.tsx (hunk 1 above)
Comment (neighborhood999) [reply static_report/src/components/ComparisonReport/ComparisonReport.tsx:345]: lack of type (`unknow`)

File: static_report/src/components/SingleReport/SingleReport.tsx (hunk 2 above)
Comment (neighborhood999) [reply static_report/src/components/SingleReport/SingleReport.tsx]: For the assertion type, we can fix it after your #253 due to schema renaming.

**Review by jonycfu — APPROVED** [review]
TODO: Fix typing gap on test assertion related components!

**Pull request merged**